shared/references/cce-sle15-avail.txt

```diff
@@ -1,4 +1,9 @@
-CCE-91398-8
+CCE-91392-1
+CCE-91393-9
+CCE-91394-7
+CCE-91395-4
+CCE-91396-2
+CCE-91397-0
```

Comment:
Why are you including new CCEs in this list?

Reply:
Hello @marcusburghardt
I will check it out.
Rumen